src/scripts/main.js

```diff
@@ -1 +1,9 @@
 'use strict';
+(() => {
+  let template = '';
+  for (let i = 0; i < 31; i++) {
+    template += '<div class="calendar__day"></div>';
+  }
+
+  document.querySelector('.calendar').innerHTML = template;
```

Comment:
It's a good practice to check if an element exists before trying to manipulate it. This can prevent potential errors if the element is not found. You can do this by storing the element in a variable and checking if it's not null before manipulating it. Here's an example: `const calendar = document.querySelector('.calendar'); if (calendar) { calendar.innerHTML = template; }`